src/__test__/venue-router.test.ts

```diff
@@ -0,0 +1,46 @@
+import request from 'supertest'
+import app from '../app'
+import { getAllVenues } from '../services/venueService'
+
+describe('GET /', () => {
```

Comment:
No need to repeat.... Option 1 - move this to another file (and remove from users), something like root-routers... or you remove from here if test already exist.

Reply:
OK